src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+
+interface Props {
+  errorMessage: string;
+}
+
+const ErrorNotification: FC<Props> = ({ errorMessage }) => {
```

Comment:
Optional - React.FC is not required

```suggestion
const ErrorNotification = ({ errorMessage }) => {
```